src/OWML.ModHelper.Input/ModInputLibrary.cs

```diff
@@ -13,22 +13,15 @@ public static class ModInputLibrary
 		public const int MaxComboLength = 7;
 		public const int GamePadKeyDiff = 20;
 
-		internal static string GetReadableMessage(RegistrationCode code)
-		{
-			switch (code)
+		internal static string GetReadableMessage(RegistrationCode code) =>
+			code switch
 			{
-				case RegistrationCode.InvalidCombination:
-					return "contains invalid keys";
-				case RegistrationCode.CombinationTooLong:
-					return "contains too many keys";
-				case RegistrationCode.CombinationTaken:
-					return "similar combination already registered";
-				case RegistrationCode.AllNormal:
-					return "correct combination";
-				default:
-					return "unknown";
-			}
-		}
+				RegistrationCode.InvalidCombination => "contains invalid keys",
+				RegistrationCode.CombinationTooLong => "contains too many keys",
+				RegistrationCode.CombinationTaken => "similar combination already registered",
+				RegistrationCode.AllNormal => "correct combination",
+				_ => "unknown"
```

Comment:
Could just replace with Enum.GetName. less lines, but doesn't explain it as well i guess...

Reply:
I agree but I prefer not to *change* Tai's stuff, as I don't really understand it. I like cleaning it up, but not actually changing the functionality :P